index.qmd

```diff
@@ -1,26 +1,48 @@
 ---
-title: "Welcome!"
+title: "Welcome to the Seedcase design documentation!"
 about:
   id: hero-heading
   template: solana
-  image: images/logo-full-color.png
-  image-width: 20em 
+  image: _extensions/seedcase-project/seedcase-theme/logos/navbar-logo-seedcase-design.svg
+  image-width: 75em 
 ---
 
 ::: {#hero-heading}
-Documentation on the architecture design of our Seedcase Products. A
-software's architecture that is well documented and described is (or
-should be) the first place to look when trying to understand it and how
-it works. The design documentation is a way of providing an overall map
-of our software (or its output) without getting into highly specific
-technical details. It is also used to communicate the goals, general
-structure, requirements, and uses of the software to both technical and
-non-technical people. Along with the design documentation, we also
-document and post the reasons
+This is the first place to look when trying to understand the Seedcase
+framework and how it works from a high-level perspective. 
+
+The design
+documentation is a way of providing an overall map of our software and
+the principles we follow when developing it without getting into highly
+specific technical details. It is also describes the goals,
+general structure, requirements, and uses of the software to both
+technical and non-technical users and stakeholders.
 :::
 
+
 {{< include /includes/_wip.qmd >}}
 
+## Contents
+
+While the Seedcase framework is software, it also interacts with data,
```

Comment:
There is something that I can't put my finger on with this sentence...

Reply:
I made a suggestion, because I agree.